StigData/Processed/WindowsServer-2012R2-DC-2.19.org.default.xml

```diff
@@ -31,7 +31,7 @@
   <!-- Ensure ''V-1172'' -ge '14'-->
   <OrganizationalSetting id="V-1172" ValueData="14" />
   <!-- Ensure ''V-1174'' -le '15'-->
-  <OrganizationalSetting id="V-1174" ValueData="15" />
+  <OrganizationalSetting id="V-1174" ValueData="14" />
```

Comment:
Any reason this was changed from 15 to 14, both values appear to be valid, but a change from the last version.

Reply:
Updated